documentation/src/docs/batch.md

```diff
@@ -0,0 +1,149 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Batch Connector
+The Flink connector library for Pravega makes it possible to use a Pravega Stream as a data source and data sink in a batch program.  See the below sections for details.
+
+## Table of Contents
+- [FlinkPravegaInputFormat](#flinkpravegainputformat)
+  - [Parameters](#parameters)
+  - [Input Stream(s)](#input-streams)
+  - [StreamCuts](#streamcuts)
+  - [Parallelism](#parallelism)
+
+- [FlinkPravegaOutputFormat](#flinkpravegaoutputformat)
+  - [Parameters](#parameters)
+  - [Output Stream](#output-stream)
+  - [Parallelism](#parallelism)
+  - [Event Routing](#event-routing)
+- [Serialization](#serialization)
+
+## FlinkPravegaInputFormat
+A Pravega Stream may be used as a data source within a Flink batch program using an instance of
+`io.pravega.connectors.flink.FlinkPravegaInputFormat`. The input format, reads a given Pravega Stream as a [`DataSet`](https://ci.apache.org/projects/flink/flink-docs-master/api/java/org/apache/flink/api/java/DataSet.html) (the basic abstraction of the Flink Batch API). Note that the stream elements are considered to be **unordered** in the batch programming model.

```

Comment:
Suggest to be more specific about the "stream elements" - events. (other things, such as streamcut may be considered as element  as well)

Reply:
reworded the statement